portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/Properties.testcase

```diff
@@ -375,6 +375,110 @@ definition {
 		}
 	}
 
+	@description = "Bug ID: LRAC-8093 | Automation ID: LRAC-11533 | Test Summary: Able to clear property data"
+	@priority = "4"
+	test ClearPropertyData {
+		task ("Launch AC and go to settings and add a new property") {
+			ACUtils.launchAC();
+
+			ACSettings.takeTokenFromAnalyticsCloud();
+		}
+
+		task ("Connect the DXP to Analytics Cloud") {
+			ACUtils.launchDXP();
```

Comment:
Hey @joshitagaki, sorry I didn't notice this earlier, but I think the file `setUp` is already doing these steps. What do you think?

Reply:
Thanks! Yea another step I forgot to remove